tests/certification/pubsub/azure/servicebus/components/authentication/service_bus.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: messagebus
+  namespace: default
+spec:
+  type: pubsub.azure.servicebus
+  version: v1
+  metadata:
+    - name: consumerID
+      value: optionscertification
+    - name: connectionString
```

Comment:
If you are testing service principal authentication you must not also specify the connection string, or else it will not actually use the service principal approach. You likely need to specify other metadata options.